saskatoon/saskatoon/settings.py

```diff
@@ -77,6 +80,7 @@
     'rest_framework',
     'django_filters',
     'crispy_forms',
+    'debug_toolbar',
```

Comment:
Maybe the debug toolbar could be enabled only if `SASKATOON_DEBUG` is True ?

Reply:
I'm pretty certain django does that by default

Reply:
ok1